PROFILE.md

```
@@ -0,0 +1,7 @@
+# This is My First Project.
+
+## I Started learning git foundation certification proejct content.
+
+### If I found any usefull information I will share and update here. 
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)